api/envoy/extensions/transport_sockets/tls/cert_mappers/static_name/v3/config.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
```

Comment:
As something possibly useful on it's own, and as an example of what can be done, want to also add another extension for this extension point that uses the SNI value as the name?